script.js

```diff
@@ -1,12 +1,183 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+const getEpisodes = document.getElementById("selectEpisodes");
+const getShow = document.getElementById("selectShow");
+const root = document.getElementById("root");
+const btn = document.getElementById("button");
+const search = document.getElementById("search");
+const allShows = [];
+let connectedEpisodes = [];
+let isShowsListingVisible = true;
```

Comment:
It's interesting to see the use of variables without using the state object to achieve the same results